content/docs/learning-guide/tutorials/environments/create-terrain-from-images.md

```diff
@@ -0,0 +1,314 @@
+
+In a heightmap, dark values are low elevation areas and light values are high elevation areas. To use a heightmap for terrain in O3DE, it must be processed as a *gradient signal image (GSI)* by [Asset Processor](/docs/user-guide/assets/asset-processor/). To ensure the heightmap is processed correctly, keep the following recommendations in mind:
+
+* 16-bit color depths are recommended. 8-bit images may not provide enough gradient steps to represent elevation, and 32-bit images might be unnecessarily large. 16-bit images can provide up to 65,536 elevation steps.
+
+* Use `.png` or `.tif` formats for 16-bit heightmap images.
+
+* Use `.tif` format for 32-bit heightmap images.
+
+* Postfix the heightmap file name with `_gsi` so that the heightmap is automatically processed by Asset Processor as a GSI (`LevelOneTerrainHeight_gsi.png`, for example). If you don't use the `_gsi` postfix, you can use [Texture Settings](/docs/user-guide/assets/texture-settings/) to configure the heightmap source asset to be processed as a GSI.
+
+{{< tip >}}
+The default **Heigh Query Resolution** for terrain in O3DE is 1.0 meters. This means a 1000 x 1000 heightmap represents one square kilometer of terrain. The **Height Query Resolution** can be changed in the **Terrain World** level component.
+{{< /tip >}}
+
+
+## Enable the terrain system
+
+First, you must enable the terrain system by adding two level components to the level prefab.
+
+1. In **O3DE Editor**, create a new level or open an existing level.
+
+    {{< tip >}}
+If your level contains the **Atom Default Environment**, delete or hide the Grid, Ground, and Shader Ball entities so that the terrain you build is not visually obstructed.
+    {{< /tip >}}
+
+2. In **Entity Outliner**, select the Level.prefab.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/enable-terrain-select.png" alt="Select the level.prefab." >}}
+
+3. In **Entity Inspector**, choose **Add Component** and add both the **Terrain World** and **Terrain World Renderer** level components to the level.prefab.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/enable-terrain-components.png" width="600" alt="Add terrain level components." >}}
+
+There are many properties available in the terrain level components that configure the terrain settings for the level. Later in the tutorial, you might need to make some adjustments to these properties.
+
+{{< important >}}
+For this tutorial, you'll set the world size of the heightmap image so that one pixel is equal to one meter. The rendering distances might need to be adjusted to make the entire terrain visible if the heightmap image is larger than the default terrain rendering distance of 4096 meters or the default camera view distance of 1024 meters.
+
+1. In the **Terrain World Renderer** level component, set the **Mesh render distance** to a value that is greater than the resolution of the heightmap, so that the entire terrain will be visible.
```

Comment:
As suggested above, I'd provide instructions and settings based on the (480 x 480) heightmap provided. Then after that, where appropriate, you can provide tips on what to do and consider when they use other heightmaps in the future.

Reply:
Most users will probably choose to use heightmaps that represents a kilometer of terrain or more. The image is a visual example.